packages/reports/addon/routes/reports/new.js

```diff
@@ -45,7 +45,13 @@ export default Route.extend({
    * @override
    * @returns {Promise} route model
    */
-  model(params, { queryParams }) {
+  model(_, transition) {
+    const queryParams =
+      (transition &&
+        (transition.queryParams || //Ember2.x support
+          transition.to.queryParams)) ||
```

Comment:
Could this fail in 2.x if `transition.queryParams` is empty? Would `transition.to.queryParams` break?